docs/dist/oauth2-redirect.html

```diff
@@ -0,0 +1,68 @@
+<!doctype html>
```

Comment:
Do you mind adding some explanation about the files under `docs/dist` in the readme? I think it would help people who want to contribute to understand about what's going on here and perhaps use the files for testing

Reply:
They are the default swagger-ui files. There's not much to change/contribute to those files (I think.)

Reply:
Perhaps about the purpose of having those files under `docs/dist`? Could they be used to display the open api file or aid the testing process? I think it would be great to add some notes about it to help someone like me who doesn't understand how useful those files are :)

Reply:
Fair enough. Added a short description in the readme. Feel free to add to it from the viewpoint of a person who isn't familiar.

done.

Reply:
Okay, thank you!